### PR TITLE
sci-mathematics/frobby: EAPI bump 4 -> 6

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

File: sci-mathematics/frobby/frobby-0.9.0-r1.ebuild [hunk 1]
```diff
@@ -1,55 +0,0 @@
...
-
-inherit eutils
-
-DESCRIPTION="Software system and project for computations with monomial ideals"
-HOMEPAGE="http://www.broune.com/frobby/"
-SRC_URI="http://www.broune.com/frobby/frobby_v${PV}.tar.gz"
-
-LICENSE="GPL-2"
-SLOT="0"
-KEYWORDS="amd64 ~arm x86"
```
Comment (Moiman) [inline sci-mathematics/frobby/frobby-0.9.0-r1.ebuild]: You are removing last stable ebuild.

Comment (SoapGentoo) [inline sci-mathematics/frobby/frobby-0.9.0-r1.ebuild]: @Moiman in `sci-*` that's fine, we don't want to make any stable guarantees anymore

File: sci-mathematics/frobby/frobby-0.9.0-r2.ebuild [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
 src_prepare() {
-	epatch \
-		"${FILESDIR}/${PN}-cflags-no-strip-soname.patch" \
-		"${FILESDIR}/${PN}-gcc-4.7.patch" \
-		"${FILESDIR}/${PN}-gmp-5.1.patch"
+	default
+
 	# CXXFLAGS are called CPPFLAGS
 	sed "s/CPPFLAGS/CXXFLAGS/" -i Makefile || die
 }
```
Comment (SoapGentoo) [inline sci-mathematics/frobby/frobby-0.9.0-r2.ebuild]: can you try `tc-export CXX` and then removing all the `CXX=` specifiers in `src_compile`?

**Comment by SoapGentoo** [comment]
@g-braeunlich I guess Macaulay needs a stable frobby, so just drop the removal commit

**Pull request closed without merging**